features/portfolio/generator.py

```
@@ -79,11 +104,24 @@ def _get_validation_errors(self, output: PortfolioOutput) -> list[str]:
 
-def _load_section_rules() -> dict[str, dict]:
+def _load_portfolio_config() -> PortfolioConfig:
     config_path = Path(__file__).resolve().parent / "config" / "portfolio.yaml"
-    with config_path.open(encoding="utf-8") as file:
-        config = yaml.safe_load(file) or {}
-    return config.get("sections", {})
+    try:
+        with config_path.open(encoding="utf-8") as file:
+            config = yaml.safe_load(file)
+    except OSError as exc:
+        raise ValueError(f"포트폴리오 설정 파일을 읽을 수 없습니다: {exc}") from exc
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)